_codux/boards/components/header/header.board.tsx

```diff
@@ -1,7 +1,7 @@
 import { createBoard } from '@wixc3/react-board';
-import ComponentWrapper from '_codux/board-wrappers/component-wrapper';
-import { CartOpenContextProvider } from '~/components/cart/cart-open-context';
-import { Header } from '~/components/header/header';
+import { ComponentWrapper } from '_codux/board-wrappers';
```

Comment:
what is _codux ?

some absolute paths start with `~` and some with `_` ?
to me this should be relative import because it is taken from same base absolute path. This is debatable though